### DISCUSSION TITLE
Установка на виртуальную машину за собственным прокси сервером который получает SSL (Synology)

### DISCUSSION THREAD

**Discussion by Vladimir-Va** [totumonline/totum-ru-issues-and-discussions#474] [Как это сделать?] [answered]
Здравствуйте, четвертый день как пытаюсь поднять тотум на своём сервере (мой уровень поднять контейнер пробросить обратный прокси до него). 
Реверс-прокси для домена `totum.online` проброшен на IP виртуального сервера для портов 80 и 443 

1. Автоустановка на Ubuntu 20 с доменом и SSL-сертификатом

Скрипт умирает на этапе проверки SSL. (вот тут вопрос почему? Ведь обратный прокси для `totum.online` настроен на 80 и 443 IP сервера totum)

2. Автоустановка на Ubuntu 20 без домена

Установка проходит успешно, по внутреннему IP адресу сервис доступен. Т.к. обратный прокси настроен, при обращении к `totum.online` вижу сообщение `Схема не найдена.`
После очередного убийства виртуалки двигаюсь в направлении настроек в Multi после no-multi.

**Comment by LostAlly** [comment, marked as answer]
Думаю у вас проблема в том, что тотум установленный на внутренний сервер имеет в названии схемы ip адрес, а при обращении через реверс прокси пытается показать схему с названием totum.online но из-за ее отсутствия говорит, что схема не найдена нужно просто ее переименовать.
Вам после проверки работоспособности работы по ip перенесите схему на доменное имя командой  bin/totum schema-replace дальше смотрите помощь по команде, я делал вручную в свое время переименованием папки и вроде изменением Какого то  файла.

**Reply by Vladimir-Va** [reply]
Спасибо, а я перепиливал конфиг Nginx. Пока получилось запустить на доменном имени, будем разбираться дальше.

**Reply by ttm-alexeypolunin** [reply]
Тотум ставится по умолчанию в режиме `multi`. В `Conf.php` в папке `/home/totum/totum-mit/` при установке прописывается какой хост к какой схеме в БД относится. Когда вы ставили его без домена, то там `ip`. Когда обращение идет через прокси (в которых я немного плаваю), то у вас уже есть домен и этот домен передается в php при запросе. Соответсвенно в Conf.php должен быть прописан домен, а не `ip`.

Если у вас были созданы поля типа Файл и в них что-то загружалось, то надо чекнуть две папки .../totum/http/fls и .../totum/secureFiles (или как-то так) там возможно вы найдете папку, которая называется как и стартовый ip. Ее надо переименовать.
